packages/grant-explorer/src/features/common/MatchingEstimateTooltip.tsx

```diff
@@ -0,0 +1,46 @@
+import { InformationCircleIcon } from "@heroicons/react/24/solid";
+import React from "react";
+import { Tooltip } from "@chakra-ui/react";
+
+export function MatchingEstimateTooltip(props: { isEligible: boolean }) {
+  return (
+    <div>
+      <Tooltip
```

Comment:
I don't know if this is a on-hover tooltip or a on-click tooltip. Since there are targets the user might be aiming at with a mouse, it might be good to ensure it's of the latter type, to avoid try-click-and-make-tooltip-disappear frustration.

Reply:
hover for mouse, click for touch. I see no targets in the vicinity of this element, but it dissapears automatically when hovered or clicked.

Reply:
I see. The target I was referring to was the link contained in the tooltip itself.